org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/ITextEditorAware.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Red Hat Inc. and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ * - Angelo ZERR (Red Hat Inc.)  - initial implementation
+ *******************************************************************************/
+package org.eclipse.ui.texteditor;
+
+/**
+ * API to inject {@link ITextEditor} instance.
```

Comment:
Please try to add documentation what an implementor can expect to happen if his random class implements the "API to inject an ITextEditor".